backend/tarsy/services/events/event_helpers.py

```
@@ -500,6 +548,9 @@ async def publish_chat_cancel_request(stage_execution_id: str) -> None:
             # Publish to cancellations channel (backend-only)
             await publish_event(session, EventChannel.CANCELLATIONS, event)
             logger.info(f"[EVENT] Published chat.cancel_requested for {stage_execution_id}")
+    except asyncio.CancelledError:
+        logger.warning(f"Event publishing cancelled (task/pod shutting down)")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)